XcodeServerSDKTests/XcodeServerTests.swift

```diff
@@ -30,6 +30,23 @@ class XcodeServerTests: XCTestCase {
     func testServerCreation() {
         XCTAssertNotNil(self.server)
     }
+    
+    // MARK: Creadentials tests
+    
+    func testCredentials() {
+        let user = server.credential?.user
+        let pass = server.credential?.password
+        
+        XCTAssertEqual(user!, "ICanCreateBots")
+        XCTAssertEqual(pass!, "superSecr3t")
```

Comment:
`host` is actually already tested in `XcodeServerConfigTests`. Nevermind me then.